eng/Versions.props

```diff
@@ -28,6 +28,7 @@
   <PropertyGroup>
     <!-- Versions used by several individual references below -->
     <RoslynDiagnosticsNugetPackageVersion>3.3.0-beta1.20262.5</RoslynDiagnosticsNugetPackageVersion>
+    <MicrosoftCodeAnalysisNetAnalyzersVersion>3.3.0-beta1.20305.4</MicrosoftCodeAnalysisNetAnalyzersVersion>
```

Comment:
I think this group of properties is only intended for ones being used by multiple different packages; I think this is only being used in the one place and should go with the larger list? (See the comment on line 29.)

Reply:
Sure, I can cleanup in a follow-up PR where I plan to bump up RoslynDiagnosticsNugetPackageVersion and fix new violations from Roslyn.Diagnostics.Analyzers.